src/api/common.ts

```diff
@@ -19,7 +19,7 @@ export type Requester<Req, Res> = (req: Req, timeout?: number) => Promise<Res>;
 /**
  * @public
  */
-export type RequesterUnit<Res> = () => Promise<Res>;
+export type RequesterNoArg<Res> = (timeout?: number) => Promise<Res>;
```

Comment:
Is this a rename from Unit (Rust) to NoArg (JS)? Now it has an arg ;-) Anyhow, not that important

Reply:
Thanks diff tool! `RequesterUnit` wasn't used so I deleted it. This one is supposed to be "no request argument"